industrial_ci/ci_main.sh

```diff
@@ -38,6 +38,15 @@
 set -e # exit script on errors
 set -x # print trace
 
+#Define some verbose env vars
+if [ "$VERBOSE_OUTPUT" ] && [ "$VERBOSE_OUTPUT" == true ]; then
+    export OPT_V="-v"
+    export OPT_I="-i"
```

Comment:
I've just would to give the possibility to use them desperately. But I see your point.